.vscode/launch.json

```diff
@@ -8,7 +8,7 @@
             "name": "Launch Chrome",
             "request": "launch",
             "type": "pwa-chrome",
-            "file": "c:\\Users\\saeed_000\\Desktop\\hyf-homework\\javascript\\javascript1"
+            "file": "c:\\Users\\saeed_000\\Desktop\\hyf-homework\\javascript"
         }
     ]
 }
```

Comment:
Do a Google search for `ignore file in git with .gitignore`. Git lets you ignore files in your folder that should not be part of the git repository. This file might be a file that should be ignored if it is only for you and not for everyone who will use this code.